benchmarks/nemo/tracer_advection/original/Makefile

```diff
@@ -46,14 +50,24 @@ tra_adv_omp: ../scripts/omp_trans.py
 	${MAKE} PROF_LIB_INC="-I../${DL_TIMER_DIR}/src" \
             LDFLAGS="${LDFLAGS} ../${DL_TIMER_DIR}/${DL_TIMER_NAME}" -C $@
 
+tra_adv_omp_gpu: ../scripts/omp_gpu_trans.py
```

Comment:
Shouldn't `./tra_adv.F90` be also one of the rule prerequesites in case it is modified it needs to be re-psycloned and recompiled?
Also in the other targets

Reply:
(Although they are .PHONY - so they are recompiled no matter what)

Reply:
Yes, you are right - done for all the targets.